basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basiclibrary;
+
+import java.util.Random;
+import java.util.Arrays;
+
+public class Library {
+//    public boolean someLibraryMethod() {
```

Comment:
it's safe to delete this once you've verified your project is up and running after your gradle init